model_compression_toolkit/core/common/mixed_precision/mixed_precision_search_manager.py

```diff
@@ -66,7 +67,10 @@ def __init__(self,
         self.layer_to_bitwidth_mapping = self.get_search_space()
         self.compute_metric_fn = self.get_sensitivity_metric()
 
-        self.compute_ru_functions = ru_functions
+        self.cuts = calc_graph_cuts(self.original_graph)
+
+        ru_types = [k for k, v in target_resource_utilization.get_resource_utilization_dict().items() if v < np.inf]
```

Comment:
change k, v to "ru_target, ru_value" or something like that, so it would be easier to understand what to expect in this list